app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+import os
+
+
+class DirectoryExistError(Exception):
+
+    def __init__(self, path: str) -> None:
+        self.path = path
+
+    def __str__(self) -> str:
+        return f"Directory with path {self.path} is already exist!"
+
+
+def move_file(command: str) -> None:
+    input_command, path_file_in, path_file_out = command.split(" ")
+
+    if input_command != "mv":
+        return
+
+    if ("/" or "\\") not in path_file_out:
+        os.rename(path_file_in, path_file_out)
+        return
```

Comment:
try to avoid using blank return

Reply:
Is it just bad practice? Do I have better option to terminate program if I want to? Like sys.exit ?

Reply:
why do u need this part of code, if 27-32 lines will make the same
u can just make a backward condition for code below, and use 27-32 lines for both conditions